gomock/controller.go

```diff
@@ -224,7 +224,7 @@ func (ctrl *Controller) Call(receiver interface{}, method string, args ...interf
 
 		expected, err := ctrl.expectedCalls.FindMatch(receiver, method, args)
 		if err != nil {
-			origin := callerInfo(2)
+			origin := callerInfo(3)
```

Comment:
Can you please add a comment about why the number three and when this value should be updated.